ra/ra.go

```diff
@@ -1758,7 +1775,8 @@ func (ra *RegistrationAuthorityImpl) RevokeCertificateWithReg(ctx context.Contex
 // called from the admin-revoker tool.
 func (ra *RegistrationAuthorityImpl) AdministrativelyRevokeCertificate(ctx context.Context, cert x509.Certificate, revocationCode revocation.Reason, user string) error {
 	serialString := core.SerialToString(cert.SerialNumber)
-	err := ra.revokeCertificate(ctx, cert, revocationCode)
+	// TODO: might want to let the comment be set via RPC?
```

Comment:
Yeah, I think we should allow setting the comment via RPC. but we could also split out the AdministrativelyRevokeCertificate changes into their own PR if you don't want to block this PR overall on that change.